nmdc_automation/re_iding/base.py

```diff
@@ -19,14 +19,20 @@
 import nmdc_schema.nmdc as nmdc
 
 from nmdc_automation.api import NmdcRuntimeApi
-from nmdc_automation.re_iding.db_utils import (OMICS_PROCESSING_SET,
-                                               READS_QC_SET,
-                                               METAGENOME_ASSEMBLY_SET,
-                                                METATRANSCRIPTOME_ACTIVITY_SET,
-                                                READ_BASED_TAXONOMY_ANALYSIS_ACTIVITY_SET,
-                                               check_for_single_omics_processing_record,
-                                               get_data_object_record_by_id,
-                                               get_omics_processing_id)
+from nmdc_automation.re_iding.db_utils import (
+    BIOSAMPLE_SET,
+    OMICS_PROCESSING_SET,
+    READS_QC_SET,
+    METAGENOME_ASSEMBLY_SET,
+    METATRANSCRIPTOME_ACTIVITY_SET,
```

Comment:
the metatranscriptome_activity_set collection contains records that are using annotations, we are deleting all of those so these should be deleted also. Perhaps have you have it is okay, in previous testing this entire collection ends up empty.